modular_skyrat/modules/reagent_forging/code/forge_weapons.dm

```diff
@@ -161,11 +147,19 @@
 	. = ..()
 	AddElement(/datum/element/kneejerk)
 
-/obj/item/forging/reagent_weapon/hammer/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
-	. = ..()
-	if(!is_type_in_list(target, fast_attacks))
+/obj/item/forging/reagent_weapon/hammer/attack(mob/living/target_mob, mob/living/user, params)
+	force = 19
+	return ..()
+
+/obj/item/forging/reagent_weapon/hammer/attack_atom(atom/attacked_atom, mob/living/user, params)
+	if(is_type_in_list(attacked_atom, fast_attacks))
+		force = 0
+		..()
+		user.changeNext_move(CLICK_CD_RAPID)
 		return
```

Comment:
Are you sure you can't just return something like `COMSIG_CANCEL_ATTACK_CHAIN` or something along those lines to make this not need to change the force like this?